spec/core_functions/selector/is_superselector/simple/pseudo/selector_arg/current.hrx

```diff
@@ -0,0 +1,80 @@
+<===> superset/input.scss
+a {b: is-superselector(":current(c d, e f, g h)", ":current(c d.i, e j f)")}
+
+<===> superset/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> subset/input.scss
+a {b: is-superselector(":current(c d.i, e j f)", ":current(c d, e f, g h)")}
+
+<===> subset/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> equal/input.scss
+a {b: is-superselector(":current(c d, e f)", ":current(c d, e f)")}
+
+<===> equal/output.css
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> bare_sub/input.scss
+a {b: is-superselector(":current(c d, e f)", "c d, e f")}
+
+<===> bare_sub/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> prefix/superset/input.scss
+a {
+  b: is-superselector(
+      ":-pfx-current(c d, e f, g h)",
+      ":-pfx-current(c d.i, e j f)");
+}
+
+<===> prefix/superset/output.css
+a {
+  b: false;
```

Comment:
Should be true here?

Reply:
Unlike almost all selector pseudos, `:current()`'s contents have to match *exactly* rather than being a superselector match. [The `:current()` pseudo](https://www.w3.org/TR/selectors-4/#the-current-pseudo) matches the currently-displayed element *if it matches the argument*, and otherwise matches the innermost parent of the argument. This means that if you broaden the set of elements matched by the argument, it can actually *stop matching* an element that it used to. So it's only safe to consider it a superselector of exactly identical selectors.